packages/smoke-test/test/commands/__snapshots__/fix.test.ts.snap

```diff
@@ -7,30 +7,27 @@ exports[`fix command > base_ts_app (file only) 1`] = `
 [STARTED] Analyzing project dependency graph
 [SUCCESS] Analyzing project dependency graph
 [STARTED] Infer Types
-[DATA] processing file: /src/get-live-neighbor-count.ts
-[DATA] processing file: /src/apply-rules.ts
-[DATA] processing file: /src/gen-random-grid.ts
 [DATA] processing file: /src/app.ts
 [TITLE] Types Inferred
-[TITLE]   12 errors caught by rehearsal
-[TITLE]   6 have been fixed by rehearsal
-[TITLE]   6 errors need to be fixed manually
-[TITLE]     -- 6 ts errors, marked by @ts-expect-error @rehearsal TODO
+[TITLE]   2 errors caught by rehearsal
+[TITLE]   0 have been fixed by rehearsal
+[TITLE]   2 errors need to be fixed manually
+[TITLE]     -- 2 ts errors, marked by @ts-expect-error @rehearsal TODO
 [TITLE]     -- 0 eslint errors, with details in the report
 [SUCCESS] Types Inferred
-[SUCCESS]   12 errors caught by rehearsal
-[SUCCESS]   6 have been fixed by rehearsal
-[SUCCESS]   6 errors need to be fixed manually
-[SUCCESS]     -- 6 ts errors, marked by @ts-expect-error @rehearsal TODO
+[SUCCESS]   2 errors caught by rehearsal
```

Comment:
looks like something is off here. these shouldn't go down.

Reply:
It was originally wrong. This is the correct output because the test is to convert a single file.